### PR TITLE
pagination

### PR DESCRIPTION
**Pull request by jthistle**
https://trello.com/c/UscxEsO3/29-paginate-the-results-list-v-ee

### CODE REVIEW

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/**](https://my-best-life-g2dbnup61-neontribe.vercel.app/) |
|:-:|
|<a href="https://my-best-life-g2dbnup61-neontribe.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/puULDFWOgR3WF13hbzntrAy8Xpc_.png">(view full size)</a> |







Commit 6c4ca706825a6b7f79d57ad9b8681e97bbc7ea38 (https://my-best-life-g2dbnup61-neontribe.vercel.app).

File: src/Components/CardList.tsx [hunk 1]
```diff
@@ -1,28 +1,123 @@
...
+  justify-content: space-around;
+  margin: 1rem 0;
+  align-items: center;
+`
+
+const NavigationButton = styled(ButtonBase)`
+  display: inline-block;
+
+  &:disabled {
+    opacity: 0;
```
Comment (Nikomus) [inline src/Components/CardList.tsx:32]: I can still tell this is here by hover 🕵️ 
I'm aiming to merge this with moving he filter here - so fine to leave for now.

File: src/Components/CardList.tsx [hunk 2]
```diff
@@ -1,28 +1,123 @@
...
 import { Card } from './Card'
 import { EmptyList } from './EmptyList'
 import { ServicePreview } from '../../pages/index'
 import { FilterContext } from '../context/FilterContext'
 import { SaveContext } from '../context/SaveContext'
+import { useRemember } from '../hooks/remember'
+import { useScrollRemember } from '../hooks/scrollRemember'
+import { ButtonBase } from './ButtonBase'
+
+const ITEMS_PER_PAGE = 20
```
Comment (Nikomus) [inline src/Components/CardList.tsx:13]: 🙌

File: src/Components/Card.tsx [hunk 3]
```diff
@@ -87,15 +87,19 @@ const SaveButtonContainer = styled.div`
...
-export const Card = ({
-  id,
-  title,
-  shortDescription,
-  image,
-  costValue,
-  costQualifier,
-  age,
-}: ServicePreview): JSX.Element => {
+// Name the function Card so it shows up in DevTools
```
Comment (Nikomus) [inline src/Components/Card.tsx:90]: I'm not sure this is making a difference for me - did you want to set the forwardRef.displayName for something?

Comment (jthistle) [reply src/Components/Card.tsx:90]: I checked; it shows up in the 'performance' tab when you take a profile, which is what I wanted.

File: src/Components/CardList.tsx (hunk 1 above)
Comment (jthistle) [reply src/Components/CardList.tsx:32]: Eh, that's probably fine. We're expecting mostly mobile users (who can't hover), and the 'disabled' state should be interpreted by screenreaders to ignore it.

**Review by Nikomus — APPROVED** [review]
Looks good to me! 📃 
I'd like to soften the styling a little to not compete with the main nav, but his can happen over in
https://trello.com/c/pR1eeDZa/92-make-the-filter-and-results-feel-more-connected-v-e

**Comment by jthistle** [comment]
Yeah, I'm a little blind when it comes to design, so this was mostly just intended to be functional. Thanks for the review!

**Pull request merged**